pennylane/ops/qubit/hamiltonian.py

```diff
@@ -550,3 +550,77 @@ def queue(self, context=qml.QueuingContext):
 
         context.append(self, owns=tuple(self.ops))
         return self
+
+    def sparse_matrix(self, wires=None):
+        """Sparse matrix representation of this Hamiltonian.
+
+        Args:
+            wires (Iterable): Wire labels that indicate the order of wires according to which the matrix
+             is constructed. If not provided, ``self.wires`` is used.
+
+        Returns:
+            :class:`pennylane.math.SparseMatrix`: sparse matrix representation
+
+        **Example:**
+
+        If no wires are given, the matrix is constructed under the assumption that the wires are ordered
+        according to the Hamiltonian's wire ordering, which is inferred from the order of the operations.
+
+        >>> ham = qml.Hamiltonian(torch.tensor([0.1, 0.2]), [qml.PauliX('b'), qml.PauliY('a')])
```

Comment:
seeing `ham` for Hamiltonian made me laugh, I don't know why 😆

Reply:
That will stick with me now too! :)